### ISSUE TITLE
NTOS Fatal Exception at 002B:C562F1B7 in TGUI -- Object doesn't support this property or method

### ISSUE THREAD

**Issue by GreyTide2108** [tgstation/tgstation#58159]
## Reproduction:

When attempting to connect to Manuel, on a fresh install of Windows 7 that _should_ have all it's drivers. on a fresh install of Byond version [513.1542], encounter this error:

A fatal exception has occurred at 002B:C562F1B7 in TGUI. The current application will be terminated. Send the copy of the following stack trace to an authorized Nanotrasen incident handler at https://github.com/tgstation/tgstation. Thank you for your cooperation.

Object doesn't support this property or method   at http://tgstation13.download/byond/assets/0c/asset.0cc3c4f09de1b94ad70afdba18353518.js:1User Agent: Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.1; WOW64; Trident/4.0; byond_5.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; Media Center PC 6.0; .NET4.0C; .NET4.0E)Nanotrasen (c) 2525-2561 

![NTOS](https://user-images.githubusercontent.com/65935938/113525297-eec5a600-9568-11eb-8233-9e16d80c4294.PNG)

![NTOSFull](https://user-images.githubusercontent.com/65935938/113525299-f08f6980-9568-11eb-8706-45f344a91bcc.png)

Attempts to disconnect and reconnect reproduce the same results.

**Comment by Serxule** [comment]
oh you're on linux arent you? that might be part of the reason

**Comment by GreyTide2108** [comment]
> 
> 
> oh you're on linux arent you? that might be part of the reason

I was. This is on a different PC. This is a Windows 7 rig. I think it has to do with my integrated graphics being bupkis.

![NoGPU](https://user-images.githubusercontent.com/65935938/113528611-ecb61400-9575-11eb-8530-f35a7b8b90ca.PNG)

**Comment by Serxule** [comment]
> I was. This is on a different PC. This is a Windows 7 rig. I think it has to do with my integrated graphics being bupkis.
yeah it likely has to do with operating system, I think people on windows 7/linux have this issue more commonly

**Comment by GreyTide2108** [comment]
I've since updated my integrated GPU drivers, I still get the same error.

![Intel HD 4600](https://user-images.githubusercontent.com/65935938/113530703-816f4080-957b-11eb-89b2-57d7754cbfa7.PNG)

**Comment by optimumtact** [comment]
what version of internet explorer is installed?

**Comment by stylemistake** [comment]
Screenshot says it's Trident 4.0 which is IE8, and it's sad to see it broke again.

**Comment by optimumtact** [comment]
they should install ie11

**Comment by GreyTide2108** [comment]
> 
> 
> they should install ie11



> 
> 
> Screenshot says it's Trident 4.0 which is IE8, and it's sad to see it broke again.

Downloads for IE 8 aren't available, and IE 11 doesn't install properly. Asks for an update but leads to a dead link. That's a me problem to work through though.

**Comment by optimumtact** [comment]
you will need to have service pack 1 of windows 7 installed, but I don't know if the windows update infrastructure is available for windows 7 anymore.

Ideally you would update to windows 10
